.circleci/config.yml

```diff
@@ -35,6 +43,8 @@ jobs:
           set -o pipefail
           [  -z "$LATEST_VERSION" ] && LATEST_VERSION="0.0.0"
           VERSION=$(echo $LATEST_VERSION | awk -F. '{$NF = $NF + 1;} 1' | sed 's/ /./g')
+          sed -i -e "s/version = \"0.0.0\"/version = \"$VERSION\"/" build.gradle.kts
```

Comment:
You need to bump the version here otherwise you'll always rebuild the same (last) version.